scripts/write_deployinst_input.py

```diff
@@ -217,63 +217,39 @@ def reactor_render(list, output_file, is_cyborg=False):
     pwr_template = read_template(template_path.replace('[reactor]', 'pwr'))
     mox_reactor_template = read_template(template_path.replace('[reactor]', 'mox'))
     candu_template = read_template(template_path.replace('[reactor]', 'candu'))
+
+    spec_dict = {}
+    # spec_dict[ 'reactor_type'] = [template, assem_size, n_assme_core(for model), n_assem_batch(for model)]
+    spec_dict['BWR'] = [pwr_template, 180, 764 / 1000, 764 / 3000]
+    spec_dict['PHWR'] = [candu_template, int(8000 / 473), 473 / 500, 60]
```

Comment:
what are all of these hard-coded numbers anyway? They should likely be variables... What's 473? 1000? 8000? 60?

Reply:
Those are values that I looked up for 'general' reactors of that type. I guess it is a bit ambiguous. Do you suggest the user input them or have a library perhaps?

Reply:
My recommendation is that they should be variables. (that is, data items with names)